src/flag_gems/ops/elu.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic(is_tensor=[True, False], promotion_methods=[(0, "DEFAULT")])
+@triton.jit
+def elu_forward_kernel(x, alpha):
+    return tl.where(x > 0, x, alpha * (tl.exp(x.to(tl.float32)) - 1))
```

Comment:
with pointwise_dynamic decorated, parameter in triton function should be treated as a scalar. so tl.where is not properly used here. is there a better choice?